src/mpp/Dec.hpp

```diff
@@ -989,9 +989,11 @@ bool jump_read(BUF& buf, T... t)
 	auto&& dst = unwrap(path_resolve(PATH{}, t...));
 	using dst_t = std::remove_reference_t<decltype(dst)>;
 	using RULE = rule_by_family_t<FAMILY>;
-	auto val = read_item<FAMILY, SUBRULE>(buf, dst);
-	if (RULE::has_children && val == 0)
-		goto decode_next_label;
+	[[maybe_unused]] auto val = read_item<FAMILY, SUBRULE>(buf, dst);
+	if constexpr (RULE::has_children) {
```

Comment:
It doesn't make sense to me - the rest of the function is about reading children. It only became a bit more complicated.

Reply:
It makes sense now, after reading the second commit - `read_item` can return type, which is not comparable with `int`. Please, write about it in the commit message.